lib/mayor_game_web/templates/city/show.html.heex

```diff
@@ -434,6 +518,14 @@
                           <hr class="border-neutral-800 flex-1" />
                           <%= to_string(value) %>
                         </div>
+                      <% else %>
+                        <%= if is_function(value, 1) do %>
+                          <div class="flex items-center gap-1">
+                            <div class=""><%= to_string(produces_type) %> per pop</div>
```

Comment:
Did you forget to include the changes in the Buildable file that define `produces_type` ?

Reply:
`produces type` comes from line 514, which was existing and derived from the produces tuple in the buildable stats.
![image](https://user-images.githubusercontent.com/22878527/218574625-1f573fb8-30fb-4fc4-a8e1-6d0305b05645.png)